core/src/main/java/io/questdb/griffin/SqlOptimiser.java

```diff
@@ -3372,6 +3361,42 @@ private QueryModel rewriteSelectClause0(final QueryModel model, SqlExecutionCont
             }
         }
 
+        if (sampleBy != null && baseModel.getTimestamp() != null){
```

Comment:
nit:
```suggestion
        if (sampleBy != null && baseModel.getTimestamp() != null) {
```

P.S. Probably we need to configure a linter finally. My built-in linter may fail from time to time.

Reply:
good spot!

I usually format code, but overlooked it this time 👍

Reply:
done in https://github.com/questdb/questdb/pull/2685/commits/1cc08eeb346ce406e7691d606ce3f06eb7436676